source/gent/tier1_hortense.rst

```diff
@@ -617,3 +633,26 @@ Getting help
 
 For questions and problems related to Tier-1 Hortense, please contact the central
 support address for Tier-1 compute: `compute@vscentrum.be <mailto:compute@vscentrum.be>`_.
+
+
+Phase 2
+-------
+
+In May 2023 a second phase was installed, adding 48 more nodes to the ``cpu_rome`` partition,
+20 extra GPU nodes with double the CPU and GPU memory in the new ``gpu_rome_a100_80`` partition and
+384 nodes using the newer AMD Milan CPUs called ``cpu_milan``.
+
+The Lustre based scratch storage was also doubled in volume and throughput to a total of 6PB.
```

Comment:
```suggestion
The Lustre based scratch storage was also doubled in volume to a total of 6PB while increasing the overal throughput as well.
```

Reply:
@itkovian the throughput was also doubled, i'll see if i can rephrase this better

Reply:
ok, nvm, using your suggestion